src/gitlab-preset.ts

```diff
@@ -0,0 +1,25 @@
+export enum GitlabRunnerPreset {
+    AMD_64_SMALL = "saas-linux-small-amd64",
+    AMD_64_MEDIUM = "saas-linux-medium-amd64",
+    AMD_64_LARGE = "saas-linux-large-amd64",
+    AMD_64_XLARGE = "saas-linux-xlarge-amd64",
+    AMD_64_2XLARGE = "saas-linux-2xlarge-amd64",
+}
+
+export const GitlabRunnerPresetValues: string[] = Object.values(GitlabRunnerPreset);
+
+export const GitlabRunnerMemoryPresetValue: Record<string, number> = {
+    [GitlabRunnerPreset.AMD_64_SMALL]: 8096,
```

Comment:
nit: personally i would prefer `8 * 1024` to save on the arithmetic

Reply:
I agree, it helps alot with the readability of the constants.